<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mcm</class>
 <widget class="QWidget" name="mcm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>乘同余法/混合同余法</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lambdlab">
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;lambda&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lambdaedt"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="mlab">
       <property name="text">
        <string>M</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="medt"/>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="x0lab">
       <property name="text">
        <string>x0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x0edt"/>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="clab">
       <property name="text">
        <string>c</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cedt"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QListView" name="listView">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QPushButton" name="surebtn">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QPushButton" name="sure2btn">
     <property name="text">
      <string>除以M</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
